dipy/reconst/sfm.py

```diff
@@ -265,13 +356,13 @@ def __init__(self, model, beta, S0, mean_signal):
             The parameters of fit to data.
         S0 : ndarray
             The mean non-diffusion-weighted signal.
-        mean_signal : ndarray
-            The mean of the diffusion-weighted signal
+        iso : ndarray
```

Comment:
Turns out this should be an `IsotropicFit` object! Now refactored in several places